apollo-federation/src/operation/mod.rs

```diff
@@ -895,7 +895,19 @@ impl Selection {
     }
 
     fn sub_selection_type_position(&self) -> Option<CompositeTypeDefinitionPosition> {
```

Comment:
This function now doesn't return the subselection's type position. I think we should rename it or inline the function where it's used (as it's only used in one place).

Reply:
Decided to inline it to contain to just that single function call. Otherwise it would have to be something like `field_sub_selection_type_position_or_fragment_type_condition` or something along those lines.